README.md

```diff
@@ -50,49 +50,52 @@ command line like this:
+Please write a static webpage that normalizes and displays the data contained in
 `sample.json`. You should write JavaScript (or code that compiles
 to JavaScript) that takes the sample data and formats it according
 to the normalization rules listed below. Don't worry about making
-it too fancy -- you can display and style the data however you wish,
+it look too fancy; you can display and style the data however you wish,
 but we will be looking for valid markup.
 
 ### Normalization rules:
 
 Normalized, in this case, means:
 
-- The entire CSV is in the UTF-8 character set.
-- The `Timestamp` column should be formatted in RFC3339 format.
-- The `Timestamp` column should be assumed to be in US/Pacific time;
+- The `Timestamp` field should be formatted in RFC3339 format.
+- The `Timestamp` field should be assumed to be in US/Pacific time;
   please convert it to US/Eastern.
 - All `ZIP` codes should be formatted as 5 digits. If there are less
   than 5 digits, assume 0 as the prefix.
-- The `FullName` column should be converted to uppercase. There will be
+- The `FullName` field should be converted to uppercase. There will be
   non-English names.
-- The `Address` column should be passed through as is, except for
-  Unicode validation. Please note there are commas in the Address
-  field; your CSV parsing will need to take that into account. Commas
-  will only be present inside a quoted string.
-- The `FooDuration` and `BarDuration` columns are in HH:MM:SS.MS
+- The `FooDuration` and `BarDuration` fields are in HH:MM:SS.MS
   format (where MS is milliseconds); please convert them to the
   total number of seconds expressed in floating point format.
   You should not round the result.
-- The `TotalDuration` column is filled with garbage data. For each
+- The `TotalDuration` field is filled with garbage data. For each
   row, please replace the value of `TotalDuration` with the sum of
   `FooDuration` and `BarDuration`.
-- The `Notes` column is free form text input by end-users; please do
-  not perform any transformations on this column. If there are invalid
-  UTF-8 characters, please replace them with the Unicode Replacement
-  Character.
+- The `Notes` field is free form text input by end-users; please do
+  not perform any transformations on this field.
+
+You can assume that the sample data we provide will contain all date
+and time format variants you will need to handle.
+
+#### For CSV solutions only
+
+- The entire data set is in the UTF-8 character set.
+- The `Address` field should be passed through as is, except for
+  Unicode validation. Please note there are commas in the Address
+  field; your CSV parsing will need to take that into account. Commas
+  will only be present inside a quoted string.
+- If there are invalid UTF-8 characters in the `Notes` field, please
+  replace them with the Unicode Replacement Character.
 
 You can assume that the input document is in UTF-8 and that any times
```

Comment:
are the above points supposed to replace this paragraph? there's some overlap -- eg utf-8. the timezone info is also mentioned above under normalization rules.

Reply:
@sojeri oh weird, I just moved that paragraph as-is from the existing README. It's confusing to me that some rules are in paragraph form vs. bullet point so maybe I should just make it consistent

Reply:
Saaaame. Thank you! <3